src/useEditable.ts

```diff
@@ -70,6 +70,9 @@ const getPosition = (element: HTMLElement): Position => {
   // Firefox Quirk: Since plaintext-only is unsupported the position
   // of the text here is retrieved via a range, rather than traversal
   // as seen in makeRange()
+  if (window.getSelection()!.rangeCount === 0) {
```

Comment:
Hm, I believe this won't be sufficient as it simply adds an "invalid" range. What I'm more curious about is why the codesandbox suddenly started having this issue as nothing should read the selection until the element is at least focused.